### PR TITLE
MINOR: Fixed couple of warnings

### PR DESCRIPTION
**Pull request by tinaselenge**
Plus some test coverage in ACL command

Co-authored-by: Mickael Maison <mickael.maison@gmail.com>
Co-authored-by: Gantigmaa Selenge  <tina.selenge@gmail.com>

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/admin/AclCommand.scala [hunk 1]
```diff
@@ -292,7 +293,7 @@ object AclCommand extends Logging {
     if (opts.options.has(opts.forceOpt))
         return true
     println(msg)
-    Console.readLine().equalsIgnoreCase("y")
```
Comment (omkreddy) [inline core/src/main/scala/kafka/admin/AclCommand.scala:295]: Can we also fix this in TopicCommand.scala?

Comment (mimaison) [reply core/src/main/scala/kafka/admin/AclCommand.scala:295]: True, we can also fix it even though in `TopicCommand` it's currently dead code as `askToProceed()` is not called.

**Comment by tinaselenge** [comment]
@omkreddy thanks for the review. We updated the TopicCommand.

**Comment by tinaselenge** [comment]
This tool is being migrated as part of https://issues.apache.org/jira/browse/KAFKA-14525.

**Pull request closed without merging**